docs/entity_alarm_control_panel.md

```diff
@@ -21,5 +39,24 @@ sidebar_label: Alarm Control Panel
 | Number | Code is a number (Shows ten-key pad on frontend).
 | Any | Code is a string.
 
+
 ## Methods
+Note, the methods also have corresponding `async_` commands (e.g., `acync_alarm_disarm`) and optinal `code` parameter.
```

Comment:
Please document it like this https://developers.home-assistant.io/docs/en/entity_switch.html

Reply:
Sure will do, but just a heads up of another way: https://developers.home-assistant.io/docs/en/entity_vacuum.html#methods

Again, perhaps generated [API-pages](http://dev-docs.home-assistant.io/en/dev/api/device_tracker.html) would be better for the entities so we can focus on good doc-strings in the code and not be have to update documentation on multiple places.